support-services/docker-compose.yml

```diff
@@ -126,7 +126,9 @@ services:
   init:
     image: alpine/curl
     # only run init if volumes were cleared
-    command: sh -c "if [ ! -f /done ]; then run-parts /docker-entrypoint.d; else echo 'Nothing to do.'; exit 0; fi"
+    command: sh -c "if [ ! -f /done ]; then run-parts /docker-entrypoint.d --exit-on-error; else echo 'Nothing to do.'; exit 0; fi"
```

Comment:
This change was simply to make sure that the init sequence would stop if any step fails.